specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2022-03-01/checkPolicyRestrictions.json

```diff
@@ -0,0 +1,408 @@
+            },
+            "description": "The check policy restrictions parameters."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The restrictions that will be placed on the resource by Azure Policy.",
+            "schema": {
+              "$ref": "#/definitions/CheckRestrictionsResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-10-01/policyMetadata.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Check policy restrictions at subscription scope": {
+            "$ref": "./examples/PolicyRestrictions_CheckAtSubscriptionScope.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/checkPolicyRestrictions": {
+      "post": {
+        "operationId": "PolicyRestrictions_CheckAtResourceGroupScope",
+        "description": "Checks what restrictions Azure Policy will place on a resource within a resource group. Use this when the resource group the resource will be created in is already known.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckRestrictionsRequest"
+            },
+            "description": "The check policy restrictions parameters."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The restrictions that will be placed on the resource by Azure Policy.",
+            "schema": {
+              "$ref": "#/definitions/CheckRestrictionsResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../stable/2019-10-01/policyMetadata.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Check policy restrictions at resource group scope": {
+            "$ref": "./examples/PolicyRestrictions_CheckAtResourceGroupScope.json"
+          }
+        }
+      }
+    },
+    "/providers/{managementGroupsNamespace}/managementGroups/{managementGroupId}/providers/Microsoft.PolicyInsights/checkPolicyRestrictions": {
```

Comment:
>managementGroupsNamespace [](http://example.com/codeflow?start=17&length=25)

I hate this but I guess its consistent with the rest of the specs for this RP

Reply:
yeah I basically copied it from some other file.
It's never too late to fix things however, lmk if you want me to make it constant (lol and then find how it breaks 10 different things down the line)